<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tasker</class>
 <widget class="QMainWindow" name="Tasker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
    <item row="0" column="2" rowspan="5">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="2" column="0">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>19</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="addButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0)</string>
      </property>
      <property name="text">
       <string>Добавить событие</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
